app/models/concerns/taxonomix.rb

```diff
@@ -35,7 +35,7 @@ def with_taxonomy_scope(loc = Location.current, org = Organization.current, inne
       self.which_location        = Location.expand(loc) if SETTINGS[:locations_enabled]
       self.which_organization    = Organization.expand(org) if SETTINGS[:organizations_enabled]
       scope = scope_by_taxable_ids(scope)
-      scope.readonly(false)
+      # scope.readonly(false) This makes the scope "incompatible" with Audit.untaxed
```

Comment:
Could youput more details here? I thino we needed to have the scope still modifiable somewhere. IIRC that was added in Rails 3 era, so the need might have changed already...